engine/env/spark/src/main/java/org/datacleaner/spark/SparkJobContext.java

```diff
@@ -212,7 +219,12 @@ private ComponentJob getComponentByKey(final AnalysisJob job, final String queri
         return null;
     }
 
-    public String getResultPath() {
+    /**
+     * Gets the path defined in the job properties file
+     * 
+     * @return
+     */
+    public String getResultPathUsercustomized() {
```

Comment:
I think I liked the previous method name better here ... what is the user customized part? Is it important for the method?